### DISCUSSION TITLE
Mail Receiving Seems to be Stucked in a Loop

### DISCUSSION THREAD

**Discussion by Acoustical** [postalserver/postal#1797] [Help with using Postal]
![1641389957(1)](https://user-images.githubusercontent.com/23265557/148226920-cee9aae8-14d7-4a6f-ae8c-70b1659e1afb.png)

I followed the installation guide. Maybe my Return Path has pointed to the mail server itself causing the problem? I don't know how to fix this.

**Comment by willpower232** [comment]
if you go into the one that hard failed, can you share a screenshot of the details on the right hand side?

**Reply by Acoustical** [reply]
![20220106100015](https://user-images.githubusercontent.com/23265557/148316084-d44dac25-708c-4993-ad34-34cd16bace28.png)

550 Loop Detected.

**Reply by willpower232** [reply]
that makes sense since your domain name has its MX record pointing at your Postal server. Not sure how you would want to deal with that.

**Reply by Acoustical** [reply]
Just want to receive emails normally.  

![DNSrecords1](https://user-images.githubusercontent.com/23265557/148495724-a26a12e0-c6f2-4af3-ba1e-c58d3e4a22fb.png)
![DNSrecords2](https://user-images.githubusercontent.com/23265557/148495730-6f9c1323-91d2-4f88-a0da-781cee1fba5d.png)

I'm a noob to this so I followed https://docs.postalserver.io/install/dns-configuration to set DNS records like this, and this problem was caused.

**Reply by willpower232** [reply]
I think this is your problem

![image](https://user-images.githubusercontent.com/1619102/148744635-e8c8b94c-571a-4b9d-90ef-de52554583f4.png)

Do you have a regular mail system with inboxes to receive mail for this domain?
